pkg/apis/core/types.go

```diff
@@ -5453,6 +5453,18 @@ type Preconditions struct {
 	UID *types.UID
 }
 
+// LogStreamType represents the desired log stream type.
+type LogStreamType string
+
+const (
+	// LogStreamTypeStdout is the stream type for stdout.
+	LogStreamTypeStdout LogStreamType = "stdout"
+	// LogStreamTypeStderr is the stream type for stderr.
+	LogStreamTypeStderr LogStreamType = "stderr"
+	// LogStreamTypeAll represents the combined stdout and stderr.
+	LogStreamTypeAll LogStreamType = "All"
```

Comment:
> Pick one style, and since things like StreamTypeStdout set precedent, I think it should be "all" or just don't define a constant for it and let "" mean all.

I totally agree. I lean more towards using "all" instead of an empty string, as it's more explicit, but I'm okay with either option. cc @SergeyKanzhelev

Reply:
Having an explicit `All` has my vote too.